src/routes/open/container/Open.tsx

```diff
@@ -81,7 +81,14 @@ export const createSafe = (values, userAccount) => {
   return promiEvent
 }
 
-const Open = ({ addSafe, network, provider, userAccount }) => {
+interface OwnProps extends RouteComponentProps {
+  userAccount: string
+  network: string
+  provider: string
+  addSafe: any
```

Comment:
this can be defined as a function, isn't it? something around `(safeProps: SafeProps) => void` not sure about the `SafeProps`, but that's the idea

Reply:
this is an action and typing an action is out of scope for this PR... I can do it, but I think there are already enough changes. Especially in our case, if you add type for 1 thing it breaks 10 things

Reply:
true that